vhdbuilder/packer/test/run-test.sh

```diff
@@ -69,7 +69,7 @@ else
   # In SIG mode, Windows VM requires admin-username and admin-password to be set,
   # otherwise 'root' is used by default but not allowed by the Windows Image. See the error image below:
   # ERROR: This user name 'root' meets the general requirements, but is specifically disallowed for this image. Please try a different value.
-  if [ ${ARCHITECTURE,,} == "arm64" ]; then
+  if [ "${ARCHITECTURE,,}" == "arm64" ]; then
```

Comment:
Just curious if there is any issue without this quote?

Reply:
noticed the error in the weekly build, but the failure was actually unrelated https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=54735158&view=logs&j=2d0bfb4e-f7ce-5220-32c3-4799f9afd656&t=ab3d9b6a-3fd0-5556-5ba7-31c2963c82da

Reply:
Yeah, should be unrelated.